coderd/database/dump.sql

```diff
@@ -1599,7 +1599,8 @@ CREATE TABLE workspace_apps (
     slug text NOT NULL,
     external boolean DEFAULT false NOT NULL,
     display_order integer DEFAULT 0 NOT NULL,
-    hidden boolean DEFAULT false NOT NULL
+    hidden boolean DEFAULT false NOT NULL,
+    open_in character varying(64) DEFAULT 'slim-window'::character varying NOT NULL
```

Comment:
Suggestion: might be better to have this as its own type similar to `workspace_app_health` or `app_sharing_level`.